src/core/server/legacy/integration_tests/logging.test.ts

```diff
@@ -87,7 +87,7 @@ describe('logging service', () => {
         const loggedString = getPlatformLogsFromMock(mockConsoleLog);
         expect(loggedString).toMatchInlineSnapshot(`
           Array [
-            "[xxxx-xx-xxTxx:xx:xx.xxxZ][INFO ][test-file] handled by NP",
+            "[xxxx-xx-xxTxx:xx:xx.xxx-xx:xx][INFO ][test-file] handled by NP",
```

Comment:
Default format changed from `ISO8601` (UTC) to `ISO8601_TZ` (retains timezone offset).